src/main.rs

```diff
@@ -25,6 +25,8 @@ use storage::content_manager::consensus::persistent::Persistent;
 use storage::content_manager::consensus_state::{ConsensusState, ConsensusStateRef};
 use storage::content_manager::toc::TableOfContent;
 use storage::dispatcher::Dispatcher;
+#[cfg(not(target_env = "msvc"))]
```

Comment:
Do we support windows and msvc compiler?

Reply:
idk, that is from the jemalloc docs

Reply:
it is better to keep this switch, I guess

Reply:
If we don't support Windows, we can just remove this checks

Reply:
what the point? If we remove and then enable the support in future, we will never going to find this place again.

Reply:
Okay, let's keep it as is. but in my opinion, we don't need any mention of windows os